locales/templates/app.pot

```diff
@@ -603,12 +607,14 @@ msgstr ""
 
 #. String displayed with an illustration to introduce a new feature
 msgctxt "onboarding_ocr_title_more"
-msgid "Do more with screenshots"
+msgid "Take Action on Text"
 msgstr ""
 
 #. String displayed to give examples of what the new feature can do
 msgctxt "onboarding_ocr_content_examples"
-msgid "Look up products, copy coupon codes, and share quotes in your screenshots"
+msgid ""
+"Search the web for products, copy coupon codes, and share quotes in your "
```

Comment:
This is a minor change but if you want to be 100% sure localizers get notified of it and retranslate accordingly, it's better to update the string entity here too.

Reply:
Same as the above.

Reply:
Usually these strings get marked as fuzzy (see https://github.com/mozilla-l10n/scryer-android-l10n/pull/10/files#diff-2270efcf5b50f573604e9f8f75128ba2R600 for example).
However here, the 2 strings I called out are not. 
Perhaps you have manually changed the files? Localizers won't get a notice this way since they won't be marked as fuzzy, unfortunately

Reply:
@Delphine I've submitted a new commit. The 3 strings we changed are marked as fuzzy in app.po file now. Thanks for noticing that!

Reply:
@benjamin-cheng Thanks this one seems good now. But I only see one string (this one) marked as fuzzy, not the one from the comment above. Maybe I'm missing something, but just in case!

Reply:
@Delphine  I've confirmed that those 3 strings we changed are marked as fuzzy in app.po file.
"onboarding_storage_content_permission" =>
https://github.com/mozilla-l10n/scryer-android-l10n/pull/10/files#diff-2270efcf5b50f573604e9f8f75128ba2R500
"onboarding_ocr_title_more" =>
https://github.com/mozilla-l10n/scryer-android-l10n/pull/10/files#diff-2270efcf5b50f573604e9f8f75128ba2R595
"onboarding_ocr_content_examples" =>
https://github.com/mozilla-l10n/scryer-android-l10n/pull/10/files#diff-2270efcf5b50f573604e9f8f75128ba2R601

Reply:
ok Thanks!